README.md

```diff
@@ -1,24 +1,151 @@
-# README
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍

Reply:
@HAMMAS-SALEEM let me record a new one, from 1:30' of the video I thought I was highlighting my favorite feature of the app, if there's any tip you can give me please, I would appreciate that. Thank you again.

Reply:
@Chaseknis Everything is perfect dear. You just need to re-record the whole video by showing all the things again and also the piece of code from your codebase 👍